index.Rmd

```diff
@@ -215,7 +215,7 @@ ggplot(time_top3,aes(x = TimeInterval, y= amount,group=1))+
   theme(legend.position = "none",axis.text.x = element_text(angle = 60, hjust = 1))
 ```
 
-## The Preference on Day of Week of Top Three Committed Crime Types
+## The Preference on Day of Week of Top Three Committed Crime Types 
```

Comment:
Awesome job! I think it could be better if you could add a period of the dataset in the title (2015-2017), add units of data on the x-axis label that would help readers easier to understand the graph.